web/src/commands/serve.ts

```diff
@@ -25,7 +28,17 @@ export async function main(argv: any): Promise<void> {
 
   metrics.bootstrapFromEnv();
 
-  await new Server(
-    argv.bugsnagKey,
```

Comment:
so we don't use bugsnag for this anymore?

Reply:
i dont think this variable was used